client/static/template.html

```diff
@@ -4,17 +4,31 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="Reversim Summit 2018">
```

Comment:
Change all these to `2019`?
```suggestion
    <meta property="og:title" content="Reversim Summit 2019">
```
etc